### ISSUE TITLE
Pagination showing even when all mods fit on one page

### ISSUE THREAD

**Issue by Thallassa** [Nexus-Mods/web-issues#68]
Example: https://i.imgur.com/LYStP1X.png there are only 40 mods (45 mods in this category plus some hidden due to the filter bug), and I'm showing 80 mods per page, so there should only be one page. However it shows 2 and 3 page buttons at the bottom and even lets me click on them (it loads the empty page as shown).

**Comment by verenion** [comment]
Can you provide a link, or steps to replicate?

**Comment by verenion** [comment]
We had an issue relating to the pagination that should have fixed this issue

**Comment by Thallassa** [comment]
The pagination issue has been fully fixed on my end, but now there's a new bug. On the same link I provided before https://rd.nexusmods.com/skyrim/mods/categories/99/, I am seeing this:

- I have it set to show 20 items per page. It is showing 15. (Telling it to show 40 and 60 show all the results as expected). 

- It says there are 40 results (there are 45 mods in this category iirc, so that's a reasonable number given my filtering). 

- It's only actually showing 32 results (no matter how many items per page I set it to show).
